### PR TITLE
DDOC-674 add a script for metadata extraction

### PR DESCRIPTION
**Pull request by bszwarc**
# Description

Add a script for metadata extraction
Preview: https://staging.developer.box.com/guides/cli/scripts/extract-metadata/

Fixes # DDOC-674

### CODE REVIEW

File: content/guides/cli/scripts/extract-metadata.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+   user.
+
+   ```bash
+   [string]$UserID = "",
+   ```
+
+   <Message>
+   If you don't specify parameters explicitly, the script
+   will prompt you to enter them.
+   </Message>
```
Comment (mgrytsai) [inline content/guides/cli/scripts/extract-metadata.md:88]: I would mention that user can either pass parameters as flags (e.g. `./Metadata-extraction.ps1 -folderId 123456789 -userId 123456789`) OR the script will prompt user for params

File: content/guides/cli/scripts/extract-metadata.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+   ```
+
+2. Specify the `folderID` of the folder you want the script to scan
+   for metadata.
+
+   ```bash
+   [string]$FolderID = "",
+   ```
+
+3. Specify the `userID` if you want to run the script on behalf of a different
```
Comment (mgrytsai) [inline content/guides/cli/scripts/extract-metadata.md:78]: Unnecessary line break (is it rendering issue)?
image

Comment (mgrytsai) [inline content/guides/cli/scripts/extract-metadata.md:78]: `Optional` parameter

Comment (bszwarc) [reply content/guides/cli/scripts/extract-metadata.md:78]: Shouldn't be there, I'll check the rendering.

File: content/guides/cli/scripts/index.md [hunk 3]
```diff
@@ -22,10 +22,12 @@ and customize.
...
+* [Extract metadata][5]
+* [Update user zones][6]
 
 [1]: g://cli/quick-start/powershell-script-templates
 [2]: g://cli/scripts/deprovision-users
 [3]: g://cli/scripts/manage-groups-collaborations
 [4]: g://cli/scripts/report-inactive-users
-[5]: g://cli/scripts/user-zones-mass-update
\ No newline at end of file
+[5]: g://cli/scripts/extract-metadata
```
Comment (mgrytsai) [inline content/guides/cli/scripts/index.md:32]: I think you need to update this part, since the file name changed
`g://cli/scripts/metadata-extraction`

**Review by mgrytsai — APPROVED** [review]
Looks great

**Pull request merged**